packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+  id="inboundcalling"
+  title="Concepts/InboundCalling"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Inbound Calling
+
+Azure Communication Services UI Library is adding support for Inbound Calling. With this new feature, users will
+be able to see and monitor new incoming calls when signed in to ACS or Teams as a Communication as a Teams User.
+Users will be able to Accept with Audio, Accept with video, and reject incoming calls.
+
+Enabling this feature will allow them see multiple incoming calls allowing the user to service multiple people
+at once best supporting the people that want to get in touch. In any scenario when your user’s are logged in you
```

Comment:
maybe "...assist multiple people simultaneously and better support those trying to reach them."